doc/application/application.rst

```diff
@@ -223,105 +621,142 @@ Below is an example CMakeList.txt:
 Application Configuration
 *************************
 
-The application's kernel is configured using a set of configuration options
+The application's kernel is configured using a set of options
```

Comment:
should this be "The Zephyr kernel..."

Reply:
This is a tough call in the presence of application-specific configuration options. The totality of the build configuration is no longer just Zephyr or application, but a combination of both.

I think this is an accurate change:

```
@@ -624,7 +624,7 @@ Below is a simple example :file:`CMakeList.txt`:
 Application Configuration
 *************************
 
-The application's kernel is configured using a set of options
+The application is configured using a set of options
 that can be customized for application-specific purposes.
 The Zephyr build system takes a configuration option's value from the first
 source in which it is specified.
```